.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 npm-debug.log
-.idea
\ No newline at end of file
+.idea
+yarn.lock
```

Comment:
I believe we should not ignore `yarn.lock`. Could you revert your `.gitignore` changes?

Reply:
Reverted